app/integrations/discord/interactions/resolvers/resolver.rb

```diff
@@ -0,0 +1,51 @@
+module Discord
+  module Interactions
+    module Resolvers
+      class Resolver
+        class << self
+          def create_resolver(request:)
+            Resolvers::Ping.new(request:) if request.ping_type?
+          end
```

Comment:
let's call this method `find` to simplify a bit. What do you think?
```suggestion
          def find(interaction:, **args)
            Resolvers::Ping.new(**args) if interaction.ping_type?
          end
```